go/contacts/tasks.py

```diff
@@ -114,10 +123,13 @@ def contacts_to_csv(contacts, include_extra=True):
 
 
 def get_group_contacts(contact_store, *groups):
+    # TODO: FIXME: Kill this thing. It keeps all contact objects in memory.
```

Comment:
The problem's actually in the callers, which take the huge list and sort it or whatever. We need to do something about those, but they're out of scope for this PR.